src/components/Accordion/FormAccordionsWithComments.tsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { ReactElement, memo, useMemo, useState } from "react";
+import { useTranslation } from "react-i18next";
+import styled from "@emotion/styled";
+import { AccordionRoot } from "@ndla/accordion";
+import { fonts, spacing } from "@ndla/core";
+import { Switch } from "@ndla/switch";
+import { IStatus } from "@ndla/types-backend/concept-api";
+import { FormAccordionProps } from "./FormAccordion";
+import OpenAllButton from "./OpenAllButton";
+import { ARCHIVED, PUBLISHED, STORED_HIDE_COMMENTS, UNPUBLISHED } from "../../constants";
+import CommentSection, { COMMENT_WIDTH, SPACING_COMMENT } from "../../containers/ArticlePage/components/CommentSection";
+import { MainContent } from "../../containers/ArticlePage/styles";
```

Comment:
Er det kun her vi bruker `MainContent` nå? Isåfall kan den flyttes

Reply:
Nope, brukes i FormAccordions og